lua/nvim-treesitter-playground/internal.lua

```diff
@@ -239,13 +312,23 @@ end
 
 M._highlight_playground_node_debounced = utils.debounce(M.highlight_playground_node_from_buffer, get_update_time)
 
-function M.get_current_node(bufnr)
+function M.get_current_entry(bufnr)
   local row, _ = unpack(api.nvim_win_get_cursor(0))
   local results = M._entries[bufnr].results
 
   if not results then return end
 
-  return results.nodes[row]
+  local node_entry = results[row]
+
+  if not node_entry then return end
```

Comment:
How can this be falsy but not nil

Reply:
It can't. I changed it to `return results and results[row]`